conbench/tests/api/test_compare.py

```diff
@@ -315,13 +315,38 @@ def test_compare_unknown_compare_ids(self, client):
         response = client.get("/api/compare/benchmarks/foo...bar/")
         self.assert_404_not_found(response)
 
+    @pytest.mark.parametrize(
+        ["baseline_result_id", "expected_z_score"],
+        [
+            # result on the fork point commit
+            (5, -2.1857497724635935),
+            # result on the parent commit
+            (6, -2.8246140882627757),
+            # result on the head commit of the default branch
+            (8, 0.12045310863100521),
+        ],
+    )
+    def test_compare_different_baselines(
```

Comment:
Okay, ready for review. This test is pretty much the meat of this PR. On `main`, the "parent" and "head of default" cases here would fail because the z-score returned looks identical to the "fork point" case z-score.

Reply:
Thank you for doing this, and for pointing out how this test would fail on main.